src/pages/AboutPage/AboutPage.js

```diff
@@ -1,22 +1,45 @@
-import React from 'react';
+import React, { PropTypes, Component } from 'react';
+import { connect } from 'react-redux';
 import { Grid, Row, Col } from 'react-bootstrap';
 import { HeroBackground, Hero } from 'components/Hero';
+import { updateDocumentTitle, resetDocumentTitle } from 'redux/modules/document-title/document-title';
 
-export default () => (
-  <section id="about-page">
-  <Hero displayUnderNavbar>
-    <HeroBackground image="/images/workspace-cb.jpg" />
-    </Hero>
-    <Grid>
-      <Row>
-        <Col xs={12} className="text-center">
-          <h1>About Page</h1>
-          <p>
-            Currently implemented as a stateless component,
-            so will not auto-update when chages are made.
-          </p>
-        </Col>
-      </Row>
-    </Grid>
-  </section>
-);
+const PAGE_TITLE = 'About Us';
+
+class AboutPage extends Component {
+  static propTypes = {
+    dispatch: PropTypes.func.isRequired,
+  };
+
+  componentDidMount() {
+    this.props.dispatch(updateDocumentTitle(PAGE_TITLE));
```

Comment:
I'd like to see this with `i18n` – so maybe it's the `messageId` that should be sent as the payload?